adds bottom margin to video in vf-content

sturobson opened the pull request.

sturobson committed:
adds bottom margin to video in vf-content

khawkins98 commented on components/vf-content/vf-content.scss:
```diff
@@ -135,6 +135,11 @@
     font-style: italic;
   }
+
+  .vf-video {
```
Might be overengineering, but should we do `.vf-video + p` as this is to be used with vf-content where we wouldn't want extra spacing at the end of section ... or before and H2/H3, we wouldn't want space there, right?

Oh, and could you drop something in the changelog about this?

sturobson replied:
ah, you've hit on something -- I guess we should cover H2/3/4/5/6 too an lists and alter accordingly.

I'll have a play and see what's a nice solution to resolve it.

sturobson replied:
… because headings have top margin as well as bottom margin (if I recall correctly)

khawkins98 replied:
👍

khawkins98 commented:
Approving. I'm indifferent on if we should tackle the spacing now or as part of a more holistic thing.

sturobson committed:
tidies up CSS output of vf-content

sturobson commented:
I have added a little bit of 'tidying up' too.

This PR also now:

- removes `display: block` from `H1-6` and `p` elements as that is their browser default
- moved `margin-bottom` changes per element to the `@include set-type` using `$custom-margin-bottom`
